server/api/core/import.js

```diff
@@ -194,12 +192,7 @@ Import.context = function (collection, selector) {
   if (selector && selector.type) {
     name = `${name}_${selector.type}`;
   }
-  // Construct a new validation context if necessary.
-  if (this._contexts[name]) {
-    return this._contexts[name];
-  }
-  this._contexts[name] = collection.simpleSchema(selector).newContext();
-  return this._contexts[name];
+  return collection.simpleSchema(selector).namedContext(name);
```

Comment:
SS already has a `namedContext` function that keeps a reference in this same way, so it seemed better to just use that